core/geometry/src/test/geometry3d/OrderedRotationAngles.test.ts

```diff
@@ -171,7 +192,8 @@ describe("OrderedRotationAngles", () => {
           ]) {
             const matrixA = anglesA.toMatrix3d();
             const anglesB = OrderedRotationAngles.createFromMatrix3d(matrixA, axisOrder);
-            testEqualOrderedRotationAngles(ck, anglesA, anglesB);
+            if (anglesB)
+              testEqualOrderedRotationAngles(ck, anglesA, anglesB);
```

Comment:
It's a good idea to assert anglesB is defined in the same step: `if (ck.testDefined(anglesB) && anglesB)`